.github/workflows/main.yml

```
@@ -11,60 +11,72 @@ jobs:
-      with:
-        submodules: recursive
-        fetch-depth: 0
-    - name: Formatting
-      uses: github/super-linter@v5
-      env:
-        VALIDATE_ALL_CODEBASE: false
-        DEFAULT_BRANCH: master
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        VALIDATE_SNAKEMAKE_SNAKEFMT: true
+      - name: Checkout with submodules
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)